flask_rebar/utils/request_utils.py

```diff
@@ -94,9 +94,22 @@ def normalize_schema(schema):
     """
     This allows for either an instance of a marshmallow.Schema or the class
     itself to be passed to functions.
+    For Marshmallow-objects support, if a Model class is passed, return its __schema__
+    Tacit assumption, if __schema__ exists, we're dealing with marshmallow_objects, caveat emptor
```

Comment:
The assumption here is kinda hack-ish, but my goal was to not introduce a hard dependency on marshmallow-objects. I went on the assumption that there is not likely to be a future version of marshmallow-objects that doesn't have `__schema__` - obvs not bulletproof but seems reasonable enough for this "bonus feature" - if a future version of marshmallow-objects were to break this, it would be trivial enough for implementing apps to pin `marshmallow-objects` accordingly (and at that point we can just note explicitly in docs that it doesn't work with those versions)

Reply:
I think it's also possible to use some import logic here:

```python
try:
    import marshmallow_objects
    def get_marshmallow_objects_schema(...):
        if isinstance(schema, marshmallow_objects.Model):
            ...
except ImportError:
    def get_marshmallow_objects_schema(...):
        ...
```

A bit hacky... but that gives you a more explicit handle on types. 
🤷  just an idea. Both approaches seem equally messy. Unfortunately, we didn't make flask-rebar very pluggable...

Reply:
you could hide those functions a little lower down, so that the def statements dont have to be indented, and then just handle trying to import two different versions of that get function as well.

Reply:
and then try to import the marshmallow-objects version, and if you get an ImportError from that since it imports marshmallow_objects, then import the current version.

Reply:
Great suggestions; I definitely like the idea of making the test for `marshmallow_objects` more explicit - will play around with this a bit before I finish off this PR.

Reply:
Updated - note that I still have to do *some* odd nesting (class def in if statement) as I added a helper class that inherits from `marshmallow_objects.NestedModel` but it does feel cleaner as that is all now contained in `marshmallow_objects_helpers`